test/e2e/scenarios/Cluster/cluster_create_update.yaml

```diff
@@ -153,16 +149,14 @@ steps:
         conditions:
           ACK.ResourceSynced:
             status: "True"
-            timeout: 7200
-
+            timeout: 180
```

Comment:
180 is a lot better than 7200, but this is essentially an instant operation too. I'd expect a much smaller timeout. Like 10 seconds.

Reply:
Will create a new PR to fix all timeout.